crates/api_common/src/utils.rs

```diff
@@ -663,6 +663,25 @@ pub async fn purge_image_posts_for_person(
   Ok(())
 }
 
+/// Delete a local_user's images
+pub async fn delete_local_user_images(
+  person_id: PersonId,
+  context: &LemmyContext,
+) -> Result<(), LemmyError> {
+  if let Ok(local_user) = LocalUserView::read_person(&mut context.pool(), person_id).await {
+    let pictrs_uploads =
+      LocalImage::get_all_by_local_user_id(&mut context.pool(), &local_user.local_user.id).await?;
+
+    // Delete their images
+    for upload in pictrs_uploads {
+      delete_image_from_pictrs(&upload.pictrs_alias, &upload.pictrs_delete_token, context)
+        .await
+        .ok();
+    }
+  }
+  Ok(())
+}
```

Comment:
Then this logic can be moved directly inside `purge_user_account()`.

Reply:
I kinda like it in its own helper function. What if I just made it non-public.

Reply:
Seems pointless if its only used in a single place, but if you insist.